src/compiler/app-core/format-component-runtime-meta.ts

```diff
@@ -2,15 +2,15 @@ import * as d from '../../declarations';
 import { CMP_FLAGS, LISTENER_FLAGS, MEMBER_FLAGS } from '@utils';
 
 
-export function formatLazyBundleRuntimeMeta(bundleId: any, cmps: d.ComponentCompilerMeta[]): d.LazyBundleRuntimeData {
+export const formatLazyBundleRuntimeMeta = (bundleId: any, cmps: d.ComponentCompilerMeta[]): d.LazyBundleRuntimeData => {
```

Comment:
this code does not run in the browser, right? why transform to arrow now?

Reply:
Not everything is going to run in the browser, but in updating the ones that are I probably updated some others too. And I guess for consistency